components/tidb_query_expr/src/impl_string.rs

```diff
@@ -702,15 +702,15 @@ pub fn instr_utf8(s: BytesRef, substr: BytesRef) -> Result<Option<Int>> {
 
 #[rpn_fn]
 #[inline]
-pub fn find_in_set(s: BytesRef, str_list: BytesRef) -> Result<Option<Int>> {
+pub fn find_in_set<C: Collator>(s: BytesRef, str_list: BytesRef) -> Result<Option<Int>> {
     if str_list.is_empty() {
         return Ok(Some(0));
     }
 
-    let s = String::from_utf8_lossy(s);
+    use std::cmp::Ordering::*;
```

Comment:
It's not necessary to import it.

Reply:
It seems that we need to, otherwise Ordering::Equal is not imported.

Reply:
you can add `use std::cmp::Ordering` to the top of the flie.